pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
@palewire again, the point IS TO HAVE CUSTOMIZED doc-strings!

when I see return value for Series.describe I want to see Series, ditto for DataFrame. that's the entire reason for shared doc-strings. *usually* we have more than 1 non-trivial parameters and so it is necessary. But we do this for all other routines, so not sure why breaking from it here. It just makes it more confusing.

This is pretty trivially to solve (for this and pct_change). Just put a doc-string with an Appender in both Series/DataFrame.

e